docs/common/security-policy.rst

```diff
@@ -7,9 +7,11 @@ In case you want to encrypt your report using PGP, please use: https://www.power
 Please do not mail security issues to public lists, nor file a ticket, unless we do not get back to you in a timely manner.
 We fully credit reporters of security issues, and respond quickly, but please allow us a reasonable timeframe to coordinate a response.
 
-We remind PowerDNS users that under the terms of the GNU General Public License, PowerDNS comes with ABSOLUTELY NO WARRANTY.
+We remind PowerDNS and dnsdist users that under the terms of the GNU General Public License, PowerDNS and dnsdist come with ABSOLUTELY NO WARRANTY.
 This license is included in this documentation.
```

Comment:
slightly offtopic, but this sentence is a lie :)

Reply:
oh, the license is missing? onward!